### DISCUSSION TITLE
[BUG]Rust Server random hard crashes

### DISCUSSION THREAD

**Discussion by YEGUnity** [GameServerManagers/LinuxGSM#3839] [Q&A]
I'm having issues with some random Rust Server crashes.  I'm not certain where to go to help trouble shoot this error, but I've attached my log below.  Any wisdom and help will be greatly appreciated.
I first contacted Facepunch but they don't support Linux and I'm not certain who makes/maintains the Linux rust server.  Thank you everyone :)

[crashlog.txt](https://github.com/GameServerManagers/LinuxGSM/files/8445090/crashlog.txt)

**Comment by h3o66** [comment]
> I'm having issues with some random Rust Server crashes. I'm not certain where to go to help trouble shoot this error, but I've attached my log below. Any wisdom and help will be greatly appreciated. I first contacted Facepunch but they don't support Linux and I'm not certain who makes/maintains the Linux rust server. Thank you everyone :)

If you open a issue please add as much info as possible (for me here is a lot of info missing).
Do they really say that they do not support their own linux server what they publish ?
That would be really strange.
Did you maybe forget the mention that you use the server ?
How did you contact facepunch ?

**Comment by YEGUnity** [comment]
I don't know what else you need for info, there is but one server that runs on linux.  I didn't think system specs would be relevant as the crash log has the details of what happened.  I'm just looking for direction for now as Facepunch Support replied as per below email chain:



This is for the server.  Who supports the server?



-------- Original message --------
From: "Holmzy (Facepunch Studios)" <support@facepunch.zendesk.com>
Subject: [Facepunch Studios] Re: Server Crash

##- Please type your reply above this line -##

Your request (224632) has been updated. To add additional comments, reply to this email.
	

Holmzy (Facepunch Studios)

Apr 6, 2022, 8:53 GMT
We do not support the Linux version of Rust

	

Tyler

Apr 6, 2022, 2:20 GMT

I can't seem to figure out why my server is crashing and need to call in for professional advice. Please see the crash log in attache filed

Attachment(s)
crashlog.txt
